core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -283,8 +436,26 @@ private static String captureStandardStream(boolean isErr, Runnable runnable) {
                 System.setErr(currentStream);
             else
                 System.setOut(currentStream);
+        }
+    }
+
+    private static class BrokerCommand implements Runnable {
```

Comment:
This can be simplified to a lambda function. For example:
```java
    private static Runnable run(Stream<String> command) {
        return () -> {
            try {
                ConfigCommand.main(command.toArray(String[]::new));
            } catch (RuntimeException e) {
                // do nothing.
            } finally {
                Exit.resetExitProcedure();
            }
        };
    }
```